prow.go

```diff
@@ -130,14 +130,39 @@ func (r *URLConfigResolver) Resolve(org, repo, branch, variant string) ([]byte,
 	}
 }
 
-// stopJob triggers namespace deletion, which will cause graceful shutdown of the
-// cluster. If this method returns nil, it is safe to consider the cluster released.
+// stopJob triggers graceful cluster teardown. If this method returns nil,
+// it is safe to consider the cluster released.
 func (m *jobManager) stopJob(name string) error {
-	namespace := fmt.Sprintf("ci-ln-%s", namespaceSafeHash(name))
-	if err := m.projectClient.ProjectV1().Projects().Delete(context.TODO(), namespace, metav1.DeleteOptions{}); err != nil && !errors.IsNotFound(err) {
+	uns, err := m.prowClient.Namespace(m.prowNamespace).Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil {
+		if !errors.IsNotFound(err) {
+			// There may have been an issue creating the prowjob; treat as success
+			return nil
```

Comment:
It still looks wierd - if we can't find the pod, we return "err can't find pod".  If get any other error, we return nil (but what error are you expecting here that would imply "we can go ahead and release it)?  User needs to retry on any error except NotFound.

Reply:
Sorry, I have to validate on my test branch which enables osd & I accidentally did a relevant push there. For real this time, removing `!` .